chainer/links/connection/grid_cells.py

```diff
@@ -0,0 +1,371 @@
+import inspect
+import six
+
+from chainer.functions.activation import lstm
+from chainer.functions.activation import sigmoid
+from chainer.functions.activation import tanh
+from chainer.functions.array import concat
+from chainer.functions.array import split_axis
+from chainer import initializers
+from chainer import link
+from chainer.links.connection import linear
+
+
+class GridLSTMBase(link.Chain):
+
+    def __init__(self, in_size, out_size,
+                 lateral_init=None):
+        super(GridLSTMBase, self).__init__(
+            lateral=linear.Linear(in_size, 4 * out_size,
+                                  initialW=0, nobias=True),
```

Comment:
In the regular LSTM the bias is removed from the lateral connection, because it is redundant with the bias from the upward connection.
We should not omit the bias here.